extensions/kubectl-cli/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "kubectl-cli-tool",
+  "displayName": "kubectl CLI Tool for Podman Desktop",
```

Comment:
Can we remove 'tool' from the name and shorten the displayName to something like `kubectl CLI`? It is redundant and long, and doesn't match other extensions (e.g. we have `Docker`, not `Docker Tool for Podman Desktop`.